docs/requirements.txt

```diff
@@ -4,4 +4,5 @@ pyyaml
 requests
 sphinx-autobuild
 sphinx-panels
+sphinx-tabs
```

Comment:
I think we could re-use the functionality that is in sphinx-panels for this, rather than adding a new dependency: https://sphinx-panels.readthedocs.io/en/latest/#tabbed-content

Reply:
Ah sorry, I'm still super new to using Sphinx directly (as opposed to Jupyter Book). If we have capacity, I'd love a sphinx directive cheatsheet for common things we include in our docs, maybe keep it in the team-compass? Another thing that **always** catches me out is that examples in the sphinx docs are usually in rST and I end up staring at my broken Markdown build for hours wondering what the problem is because I copy-pasta'd.

Reply:
Done in [`d449468`](https://github.com/2i2c-org/infrastructure/pull/836/commits/d449468536dfdd9afeca00a49777ec4f0ab71e57)

Reply:
@sgibson91 that's a cool idea re: common directives. Maybe we can add something to the team compass that just has a list of stuff that is useful?

Reply:
I would love that!